studio/components/interfaces/Organization/TeamSettings/MembersView.tsx

```diff
@@ -126,13 +126,17 @@ const MembersView = () => {
                               <span className="flex rounded-full border-2 border-border-secondary-light p-2 dark:border-border-secondary-dark">
                                 <IconUser size={20} strokeWidth={2} />
                               </span>
-                            ) : (
+                            ) : x.username.indexOf('@') < 0 ? (
```

Comment:
should we use a better check here. can we actually check what provider the user is?
this looks like it could break easily when we add new providers

@joshenlim @alaister

Reply:
yup - definitely, this has actually been fixed now in https://github.com/supabase/supabase/pull/11033